confighelper/__init__.py

```diff
@@ -13,12 +13,13 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""confighelper is a support module for Parkbench projects. Currently it only holds a class
-to help read and validate configuration files."""
+# TODO: Rename this package to parkbench-common.
+"""confighelper is a support module for Parkbench projects."""
 
+# TODO: This isn't done correctly, but removing it will break other projects right now.
 from .confighelper import ConfigHelper
 from .confighelper import ValidationException
 
-__all__ = ['ConfigHelper', 'ValidationException']
+__all__ = ['beaconbroadcaster', 'beaconreceiver', 'confighelper', 'tmpfs']
```

Comment:
I'd probably rename beaconreceiver to beaconbroadcastconsumer. In the future, there will be beaconmessagesenders and beaconmessagereceivers (or something like that).

Reply:
I'm sorry, did I stumble into Java land? :stuck_out_tongue: